src/style-sheets/layout/_toolbar.scss

```diff
@@ -45,6 +45,9 @@
 
       & img {
         width: 148px;
+        @include medium("down"){
```

Comment:
there's a medium media query down at the bottom already that you could add this to. I do kinda like the idea of the media query being with where the change is being made. So we could discuss that if that's a common practice you like to follow